src/services/lesson/index.js

```diff
@@ -21,6 +21,8 @@ module.exports = function setup() {
 	app.use('/lessons/:lessonId/files', new LessonFilesService());
 	app.use('/lessons/copy', new LessonCopyService(app));
 
+	app.use('/lessons/:lessonId/material', new AddMaterialService());
```

Comment:
where are the auth/right checks (hooks) for this implemented?

Reply:
they are implemented in hooks/addMaterial.js